server/start.go

```diff
@@ -77,6 +78,7 @@ which accepts a path for the resulting pprof file.
 		FlagMinGasPrices, "",
 		"Minimum gas prices to accept for transactions; Any fee in a tx must meet this minimum (e.g. 0.01photino;0.0001stake)",
 	)
+	cmd.Flags().IntSlice(FlagUnsafeSkipUpgrade, []int{-1}, "Skip current software upgrade to continue the old binary")
```

Comment:
What about simply using `[]int{}` here? We don't need a flag value, and I would assume this default if no flags are passed.

Reply:
That should be better, will use empty array as default